### DISCUSSION TITLE
Cytoscape canvas + grid behavior request

### DISCUSSION THREAD

**Discussion by Chadha93** [cytoscape/cytoscape.js#3040] [Q&A]
Hey,

I want to control the scroll behavior of the canvas in the following ways:

1. Is there a way to enable vertical and horizontal scroll while keeping canvas dragging enabled? 
2. The Cytoscape grid can be used to add a grid to the canvas. Is there a way to change that grid to another dynamic style such as dynamic dots similar to FigJam?

Thank you

**Comment by Chadha93** [comment]
Hi, @maxkfranz is this something achievable with the current cytoscape.js? 
Thank you.

**Comment by Abhishek-kumar09** [comment]
Probably an answer to 1st question:
1. https://stackoverflow.com/questions/27404879/vertically-scrolling-graph-with-fixed-size-nodes-with-cytoscape-js

Is there any alternate or better way of doing this. // @maxkfranz
